tools/check_format_compatible.sh

```diff
@@ -101,7 +102,7 @@ done
 for checkout_obj in "${forward_compatible_checkout_objs[@]}"
 do
    echo == Build "$checkout_obj" and try to open DB generated using $checkout_flag...
-   https_proxy="fwdproxy:8080" git checkout github_origin/$checkout_obj -b $checkout_obj
+   git checkout $checkout_obj
```

Comment:
The local branch is already checked out in previous step. no need to redo that here.